hacs.json

```diff
@@ -1,4 +1,5 @@
 {
-  "name": "Lovelace Grocy Chores Card",
-  "render_readme": true
-}
\ No newline at end of file
+  "name": "Lovelace Grocy Chores Card (Packaged)",
```

Comment:
Probably want to revert back to the original hacs name, I modified it so that I could differentiate between this and the original in HACS.